Compiler warning

Windows build currently has this warning:

```
src/run.c:7:39: warning: unused parameter 'valgrind' [-Wunused-parameter]
int run_exe(const char *exepath, bool valgrind)
                                      ^
1 warning generated.
```

This kinda makes sense because valgrind doesn't run on Windows, but the warning should be silenced if on Windows.

--- a/bootstrap_compiler/run.c
+++ b/bootstrap_compiler/run.c
@@ -8,6 +8,7 @@ int run_exe(const char *exepath, bool valgrind)
 {
     char *command = malloc(strlen(exepath) + 1000);
 #ifdef _WIN32
+    assert(!valgrind);
     sprintf(command, "\"%s\"", exepath);
     char *p;
     while ((p = strchr(command, '/')))

